playground/assets/index.html

```diff
@@ -71,7 +71,8 @@ <h2>CSS url references</h2>
     CSS background image-set() (with multiple descriptor)
   </span>
 </div>
-<div
+<!-- image-set in style tags is not supported in Vite yet -->
+<!-- <div
   style="
     background-image: -webkit-image-set(
       './nested/asset.png' type('image/png') 1x,
```

Comment:
We currently don't support `image-set` in style tags, only `url()`, which we can improve later. This was causing 404s in the latest chromium browser as it now understands the syntax, but the 404s cause the test to fail.